dist/community-toolbox.js

```diff
@@ -82232,6 +82232,7 @@ function insertIssue(issue, el) {
     body += "<a class='label label-default' href='" + label.url + "' style='background:#" + label.color + ";'>" + label.name + "</a> ";
   });
   body += "</div>";
+  body += "<img width='50px' src='" + issue.user.avatar_url + "' class='img-circle'> &emsp;";
```

Comment:
Ah, ok so this actually needs to go in the src directory, in the corresponding file there, but then when you run "grunt build" itll get compiled into the dist folder. Does that make sense? Thank you!